modules/ztp-disconnected-environment-prereqs.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-disconnected-environment-prereqs_{context}"]
+= Prerequisites
+
+You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2] in the location that will host the {product-title} cluster, such as one of the following registries:
```

Comment:
```suggestion
You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2] in the location that can host the {product-title} cluster, such as one of the following registries:
```

Reply:
I'm not going to implement this one. I think we need to be impertive here, so "will" works in this instance.